src/main/java/net/mcreator/element/types/Plant.java

```diff
@@ -88,6 +88,9 @@
 	public int flammability;
 	public int fireSpreadSpeed;
 
+	public List<MItemBlock> canBePlacedOn;

```

Comment:
This can now be null for existing elements, so initialize this to an arraylist inside Plant() constructor, similar to how some other mod element types do it in similar cases.

Reply:
Done.